earth_enterprise/src/fusion/autoingest/Asset.h

```diff
@@ -38,7 +38,7 @@
  ***
  ******************************************************************************/
 class AssetImpl : public AssetStorage, public StorageManaged {
-  friend class AssetHandle_<AssetImpl>;
+  friend class AssetHandle_<AssetImpl, AssetStorage>;
```

Comment:
Minor: you might be able to get away from all of these extra template parameters by adding `using Base = AssetStorage;` in this class and then referring to AssetType::Base in the storage manager.

Reply:
I like that. I wasn't super happy about adding the extra template parameter. It's changed now.